pydantic/main.py

```diff
@@ -815,6 +817,24 @@ def _get_value(
         else:
             return v
 
+    @classmethod
+    def try_update_forward_refs(cls, **localns: Any) -> None:
+        """
+        Same as update_forward_refs but will not raise exception
+        when forward references are not defined.
+        """
+        if cls.__module__ in sys.modules:
+            globalns = sys.modules[cls.__module__].__dict__.copy()
+        else:
```

Comment:
when is it the case?

Reply:
Without it, some of the tests are failing.